### DISCUSSION TITLE
Filament form repeater as table

### DISCUSSION THREAD

**Discussion by medfrk** [filamentphp/filament#10455] [Help]
How to create a repeater that looks like **key-value**  field but with components: Select, TextInput, ... :

<img width="1690" alt="Screenshot 2023-12-23 at 13 12 57" src="https://github.com/filamentphp/filament/assets/2337817/d21624a8-c5ff-4748-b3e1-6649a61ec1ca">

**Comment by leandrocfe** [comment]
Maybe table repeater?
https://filamentphp.com/plugins/awcodes-table-repeater

**Reply by medfrk** [reply]
Thank you very much i tested awcodes-table-repeater and it's works very good
